src/content/intersectors.ts

```diff
@@ -83,7 +80,7 @@ const mutationObserver = new MutationObserver(async (mutationList) => {
 // We observe document instead of document.body in case the body gets replaced
 mutationObserver.observe(document, config);
 
-function addIntersector(element: Element) {
+function addIntersectorObserver(element: Element) {
```

Comment:
I'm not sure about the name. I wanted to change the name of this function because it didn't really match what the function does, but I think something like `observeIntersectionIfClickableOrText` would be more descriptive. A bit long, I know

Reply:
Something like `maybeObserveIntersection` could also work